types/src/config.rs

```diff
@@ -1,16 +1,16 @@
+use crate::IntentsAccount;
+use crate::date_time;
+use crate::discount::Discount;
+use crate::duration::Duration;
+use crate::utils::{is_all_unique, to_u128};
+use alloy_primitives::ruint::aliases::U256;
 use near_sdk::json_types::U128;
 use near_sdk::serde::de::Error;
 use near_sdk::serde::{Deserialize, Deserializer, Serialize, Serializer};
 use near_sdk::{AccountId, near};
 use std::fmt::{Display, Formatter};
 use std::str::FromStr;
 
-use crate::IntentsAccount;
-use crate::date_time;
-use crate::discount::Discount;
-use crate::duration::Duration;
-use crate::utils::is_all_unique;
```

Comment:
Let's leave it as it was before.

Reply:
In this case I did nothing. This is entirely the IDE’s initiative.

Reply:
I understand it, but could you move it back?